page.go

```diff
@@ -444,27 +444,35 @@ func (page *repeatedPage) Slice(i, j int64) BufferedPage {
 		panic(errPageBoundsOutOfRange(i, j, numRows))
 	}
 
-	rowIndex0 := int64(0)
-	rowIndex1 := int64(len(page.repetitionLevels))
-	rowIndex2 := int64(len(page.repetitionLevels))
+	rowIndex0 := 0
+	rowIndex1 := len(page.repetitionLevels)
+	rowIndex2 := len(page.repetitionLevels)
```

Comment:
My first thought is that this is a typo since you're assigning two things to the same value where you might expect the second to be definitionLevels or whatever instead. Not sure how to make this clearer.

Reply:
The idea is to set the values to the maximum, which would be the value we would land on if `i = j = numValues`.